src/analytics/selectors.test.ts

```diff
@@ -188,8 +251,14 @@ describe('getCurrentUserTraits', () => {
       phoneCountryCallingCode: '+33',
       phoneCountryCodeAlpha2: 'FR',
       pincodeType: 'CustomPin',
+      positionsAppsCount: 4,
+      positionsCount: 5,
+      positionsTopTenApps: 'b:71.11,a:10.00,c:2.22,d:0.01',
       tokenCount: 13,
+      topTenPositions:
+        'b-Title E:70.00,a-Title A:10.00,c-Title C:2.22,b-Title B:1.11,d-Title D:0.01',
```

Comment:
I'm a bit torn on including the title here.
I thought it would be simpler for us to understand what the position is, but in the future it will probably be localized.
Let me know what you think.